core/README.md

```diff
@@ -81,16 +66,31 @@ that declare different conformance classes. This is useful when an entire catalo
 support the *STAC API - Item Search* conformance class, perhaps because it uses multiple databases to store items,
 but sub-catalogs whose items are all in one database can support search.
 
-A STAC API conforming to the *STAC API - Browseable* conformance class
-(<https://api.stacspec.org/v1.0.0-rc.2/browseable>) must be structured such that
-all Items in the catalog can be accessed by following `child` and `item` link relations. This is a more significant
-constraint than a STAC API without this conformance class or a STAC Catalog that is available over HTTP but does not
-implement STAC API, neither of which have any guarantee regarding the reachability of Items. This conformance 
-class is used to signal to users that they can fully navigate to all available Items using a UI (like [STAC Browser](https://github.com/radiantearth/stac-browser), 
-and also makes it clear to crawlers that they can reach everything by following catalog links. 
+## Browseable
+
+The Browseable conformance class (<https://api.stacspec.org/v1.0.0-rc.2/browseable>) provides a formal way
+to advertise that all Items in the Catalog presented by a STAC API may be reached by following `child` and
+`item` link relations. In a non-API STAC Catalog, all items must be reachable via these relations to be
+considered in the catalog. With a STAC API Catalog, items are considered to be in the catalog if they are
+accessible via search operations. However, a STAC API Catalog may also have child and item link relations.
+This presents an ambiguity between whether the same set of items accessible via search operations
+is also accessible via link relations. Browseable clarifies this by unambiguously advertising that
+items available via search can also be accessed by following link relations.
+
+Support for this "browse" mode of interaction via link relations is complementary to the dynamic search
+capabilities defined in the *STAC API - Features* and *STAC API - Item Search* conformance classes.
```

Comment:
Maybe put links to the features and item searches? Just so people can easily get to them to comapare.

Reply:
done